auslib/blobs/apprelease.py

```diff
@@ -75,6 +75,14 @@ def getBuildID(self, platform, locale):
                 raise BadDataError("No buildID for platform '%s'" % (platform))
             return self['platforms'][platform]['buildID']
 
+    def has_wnp(self, field):
+        # check for What's new page for Blobs < v9
+        for group in self.get(field, []):
+            if isinstance(group, dict) and '/whatsnew/' in group['fields'].get('openURL', ''):
```

Comment:
This can be simpler, actually - you just need to look for whether or not "openURL" exists at all. It doesn't matter what the URL is, if it's set, it's a WNP! I think it would be better to have separate implementations of this for V9 and <V9, too. They're different enough that trying to make them all work in one function is pretty ugly.